docs/l3/api_flow.md

```diff
@@ -525,4 +525,106 @@ Note: A demand can be commented on multiple times and it does not change the dem
   }
 ```
 
-Note: At any time after a `match2` is proposed, and before it reaches `acceptedFinal` state, any of its members can reject the `match2` using `POST /v1/match2/{id}/rejection`. Once a `match2` is rejected, it is permanently closed.
\ No newline at end of file
+Note: At any time after a `match2` is proposed, and before it reaches `acceptedFinal` state, any of its members can reject the `match2` using `POST /v1/match2/{id}/rejection`. Once a `match2` is rejected, it is permanently closed.
+
```

Comment:
```suggestion

When a `match2` is in a state of `acceptedFinal` it can be cancelled by either Member A or Member B, if seen fit i.e. order or capacity is no longer available for a specific reason.

1. For example, cancelling `match2` as Member A using `POST /v1/match2/{match2Id}]/cancellation`:

___remove___```js
{
    "id": "1d007432-519b-41ca-8d36-ded5b3d40d52",
    "state": "submitted",
    "localId": "fef4a34b-e3a2-46bb-b4a0-c67cc32eac94",
    "apiType": "match2",
    "transactionType": "cancellation",
    "submittedAt": "2023-09-27T15:13:18.464Z",
    "updatedAt": "2023-09-27T15:13:18.464Z"
}
___remove__```

```